RandomizerLib3.0/RandoSettings.cs

```diff
@@ -177,8 +178,74 @@ public RandoSettings Clone()
             result.FireballSkips = FireballSkips;
             result.DarkRooms = DarkRooms;
             result.Seed = Seed;
+            result.GeoSeed = GeoSeed;
 
             return result;
         }
+
+        public int GetSettingsSeed()
```

Comment:
First time I've seen this kind of implementation for a bitmask setting function, I really do think we should use constant bits rather than a template of `if(...) v +=1; v<<1`.  It would be more efficient and much cleaner